src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -0,0 +1,167 @@
+{
+  "title": "FV3 Forecast config",
+  "description": "This document is to validate user-defined FV3 forecast config files",
+  "type": "object",
+  "properties": {
+    "platform": {
+      "description": "attributes of the platform",
+      "type": "object",
+      "properties": {
+        "WORKFLOW_MANAGER": {
+          "type": "string",
+          "enum": [
+            "rocoto",
+            "none"
+          ]
+        },
+        "NCORES_PER_NODE": {
+          "type": "number"
+        },
+        "SCHED": {
+          "type": "string",
+          "enum": [
+            "slurm",
+            "pbspro",
+            "lsf",
+            "lsfcray",
+            "none"
+          ]
+        },
+        "CCPA_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "MRMS_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "NDAS_OBS_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "METPLUS_PATH": {
+          "type": "string",
+          "format": "uri"
+        },
+        "MET_BIN_EXEC": {
+          "type": "string"
+        },
+        "MET_INSTALL_DIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "DOMAIN_PREGEN_BASEDIR": {
+          "type": "string",
+          "format": "uri"
+        },
+        "PARTITION_DEFAULT": {
```

Comment:
Partitions and queues should be specified, as we'd discussed before in the arguments to the batch card or run command creator. These are all bad constructs invented by SRW that we shouldn't be enforcing.